sunyar-fabric (#216)
hyperledger-labs/hyperledger-labs.github.io: sunyar-fabric.md

---
layout: default
title: Sunyar fabric
parent: Labs
---
# Lab Name
Sunyar fabric

# Short Description

> Sunyar is a settlement distributed system based on Hyperledger fabric. This is a non-profit project for contributing charities to help needy people in a decentralized ecosystem.

Reviewer: ```suggestion
Sunyar is a settlement distributed system based on Hyperledger Fabric. This is a non-profit project for contributing charities to help needy people in a decentralized ecosystem.
```

Reviewer: Please note that there is another lab focused on charity tracking and Hyperledger Fabric -- [Karma Charity Platform](https://labs.hyperledger.org/labs/karma-charity-platform.html). This might be a good lab to collaborate with for ongoing work.

Author: Sounds great. we are trying to read their project and contribute on required situations.

# Scope of Lab

> Sunyar is a settlement distributed system based on Hyperledger fabric. There are several usecased in banking usecased in settlement distributed systems. Our goal is to achieve an appropriate project on hyperledger fabric to solve other challenges and usecases on banking systems.

Reviewer: ```suggestion
Sunyar is a settlement distributed system based on Hyperledger Fabric. There are several use cases in banking. Our goal is to achieve an appropriate project on Hyperledger Fabric to solve other challenges and use cases on banking systems.
```

Reviewer: From the README in the repo linked below, there is the following description:

> Sunyar is a distributed settlement platform, which can be used by charities for managing donation process. This can be done by an open-source platform, which can link to a distributed ledger for charities. The open-source platform helps them for automating their process and the distributed ledger can used for two main reasons: uniqueness of needy and their requirements, and transparent aid for them).

Based on the scope given above, it looks like the intent is to go beyond just the charity aspect. Is this true?

Author: Actually our Lab has focused on distributed settlement to accomplish other use cases in Fintech and banking systems such as Post-trade securities and Cross border payments.

# Initial Committers
- https://github.com/sunyar-fabric
- https://github.com/sanazsasan
- https://github.com/saman136757
- https://github.com/mortezamahdavimz
- https://github.com/mmdhossein
- https://github.com/ZankoMohammadzadeh

# Sponsor
_Provide the name of your sponsor. A sponsor is optional, but the sponsor must be a maintainer of one of the Hyperledger projects, a TSC member, or a WG chair. Read about sponsors' duty in [Section 3, Labs proposal](./index.md#process-to-propose-a-new-lab)._
- https://github.com/<user_id> or Name (email@example.org) - Role (e.g., "Chair of the XXX workgroup")

# Pre-existing repository
_If you currently have a Github repository that you wish to transfer to the Hyperledger Labs organization, please provide a link here. **NOTE: Please refer to the README for additional information on existing repositories.**_

> •	https://github.com/sunyar-fabric/sunyar

Author: thank you for your review and effort.
Do you mean amending existing commits of "hyperledger-labs.github.io" repository?

> •	https://github.com/sunyar-fabric/sunyar

Author: So, I should close this fork, and create another fork?

> •	https://github.com/sunyar-fabric/sunyar

Reviewer: For option 1, you would be amending the commits in https://github.com/sunyar-fabric/sunyar. See https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md. A quick solution to signoff last X number of commits.

```
git rebase --signoff @~X
```

For option 2, we would create the new lab repo and then you would manually transfer your code from the existing repo into this newly created lab repo.

Your choice.

> •	https://github.com/sunyar-fabric/sunyar

Reviewer: I see that there are still some commits that have not been signed off.

> •	https://github.com/sunyar-fabric/sunyar

Author: I cloned the repository into a directory. 
I committed this command.
$ git rebase -i HEAD~20
then I pushed forced to the repository again.
please check the DCO, commits have been signed off.
thank you

---
layout: default
title: Sunyar

> [//]: # (parent: Labs)

Reviewer: ```suggestion
parent: Labs
```

---
# Lab Name

> POLWINNO Innovation Center 

Reviewer: The lab name is supposed to be the same as the name of the .md file.
```suggestion
Sunyar Fabric
```

> # Short Description

Reviewer: This should be a short description of the GitHub repo that would be created for this lab. In other words, describe in a sentence the lab Sunyar Fabric.

POLWINNO innovation center is committed to drive collaboration and innovations in the financial services industry. POLWINNO is affiliated by Behsazan Farda Holding; an IT company in banking ecosystem. Provide a short description of your lab. This will be used for the Github repository's description.

# Scope of Lab

> Sunyar is a charity project but polwinno's main goal is to distributed settlement.

Reviewer: This should be the scope of what you are intending to create with this lab. As mentioned in the template, please provide enough information for us to understand how this fits with Hyperledger's mission.

> 

Reviewer: We need names of the people who will be contributing to this lab. Without that, no one will have write access to commit to the GitHub repo.

# Pre-existing repository
_If you currently have a Github repository that you wish to transfer to the Hyperledger Labs organization, please provide a link here. **NOTE: Please refer to the README for additional information on existing repositories.**_

> •	https://github.com/sunyar-fabric/sunyar

Reviewer: I received a 404 when attempting to look at this repo in GitHub. Please make the repo public so that we may check things such as license and DCO sign offs. This will be required before we allow for this lab to be approved.
